.gitignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+node_modules/**/*
```

Comment:
you can change the two lines into a single one `/node_modules`.

Reply:
Hello @ChrisZie

You could also ignore `.DS_Store`.